libs/plugins/stream-client/src/lib/params-schema/params-schema.component.html

```diff
@@ -0,0 +1,151 @@
+<bs-modal [size]="'lg'" #modal cssClass="flow-input-params-modal">
```

Comment:
can we replace the bs-modal with the modal library? also can we rename the `flow` classes to `streams`?

Reply:
I think it's better to make the switch to the modal lib in another PR, this one is already extensive enough